src/pywrangler/utils.py

```diff
@@ -79,20 +79,30 @@ def run_command(
     """
     logger.log(RUNNING_LEVEL, f"{' '.join(str(arg) for arg in command)}")
     try:
```

Comment:
Maybe:
```py
kwargs = {}
if capture_output:
    kwargs = dict(
        stdout=subprocess.PIPE,
        stderr=subprocess.STDOUT
    )
```
and then add `**kwargs` to the `subprocess.run()` call.

Reply:
I tried this with `{"stdout": subprocess.PIPE, "stderr": subprocess.STDOUT}` but mypy didn't like it. I guess `dict` may work though?

Reply:
I think a `type: ignore` is better than duplicating the call.